### PR TITLE
Implemented recommended changes

### PR DESCRIPTION
**Pull request by krutideepanpanda**
# Description

Changed to allow changing user fields in admin console.

Fixes # (issue)
- made change to MyUserAdmin class
- set Debug as true in settings.py
## Dependencies

List any dependencies that are required for this change.

## Type of Change

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Lol, trusted Nishant on this.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: embedathon/admin.py [hunk 1]
```diff
@@ -65,10 +65,14 @@ class Meta:
         model = User
         fields = ('id', 'username', 'first_name', 'last_name', 'email', 'is_staff', 'phone', 'ieee_number', 'college_name', 'team_leader', 'team_member')
 
-class MyUserAdmin(ExportMixin, UserAdmin):
```
Comment (nishant-nayak) [inline embedathon/admin.py:68]: Can you keep this in? This helps us export details of the users in CSV format

File: embedathon/admin.py [hunk 2]
```diff
@@ -65,10 +65,14 @@ class Meta:
...
-    list_display = ("username", "email", "first_name", "last_name", "phone", "college_name", "ieee_number", "is_staff")
+class MyUserAdmin(UserAdmin):
+    list_display = ("username", "email", "first_name", "last_name", "phone", "college_name", "ieee_number")
     list_filter = ("is_staff", "is_nitk", "is_active", "groups")
-    resource_classes = (UserResource,)
```
Comment (nishant-nayak) [inline embedathon/admin.py:71]: Can you also keep this in? Relates to the exporting of user data

**Review by nishant-nayak — CHANGES_REQUESTED** [review]
Requested some changes in `admin.py`. `settings.py` looks good.

**Comment by krutideepanpanda** [comment]
Changes have been implemented. Thanks for the feedback.

**Review by nishant-nayak — APPROVED** [review]
LGTM!

**Pull request merged**